install.sh

```diff
@@ -453,6 +442,9 @@ install_airgap_tarball() {
         info "decompressing airgap tarball to ${INSTALL_RKE2_AGENT_IMAGES_DIR}"
         gzip -dc "${TMP_AIRGAP_TARBALL}" > "${INSTALL_RKE2_AGENT_IMAGES_DIR}/rke2-images.${SUFFIX}.tar"
     fi
+    # Search for and install additonal rke2 images
+    find "${INSTALL_RKE2_ARTIFACT_PATH}" -type f -name "rke2-images-*.${SUFFIX}*" \
```

Comment:
The install script doesn't currently require findutils be installed on the host. Can we do this with just a `for IMAGE in ${INSTALL_RKE2_ARTIFACT_PATH}/rke2-images-*.${SUFFIX}*; do cp -v "${IMAGE}" "${INSTALL_RKE2_AGENT_IMAGES_DIR}" ; done` loop instead?

Reply:
Sure